client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -185,6 +193,18 @@ protected RestHighLevelClient(RestClient restClient, List<NamedXContentRegistry.
     }
 
     /**
+     * Returns the low-level client that the current high-level client instance is using to perform requests
+     */
+    public RestClient getLowLevelClient() {
+        return client;
+    }
+
+    @Override
+    public void close() throws IOException {
+        client.close();
+    }
```

Comment:
If its not complicated, maybe add a test that HL close also closes the LL client and is idempotent (as of last change requested by Ryan).